test/perf.sh

```diff
@@ -1,76 +1,151 @@
 #!/bin/bash
+
 set -e
-cd "$(dirname "$0")"
+set -o pipefail
 
-TIME=${2:-30}
+cd "$(dirname "$0")"
 
 server_address() {
-    limactl shell server ip -j -4 addr show dev lima0 | jq -r '.[0].addr_info[0].local'
+    limactl shell perf1 ip -j -4 addr show dev lima0 | jq -r '.[0].addr_info[0].local'
 }
 
 create() {
-    limactl create --name server --tty=false lima.yaml &
-    limactl create --name client --tty=false lima.yaml &
+    echo "[perf] Creating $count vms"
+    for i in $(seq $count); do
+        limactl create --name perf$i --tty=false lima.yaml &
+    done
     wait
 }
 
-host-to-vm() {
-    limactl start server
-    limactl shell server sudo systemctl start iperf3.service
-    iperf3-darwin --client $(server_address) --time $TIME
-    limactl stop server
+delete() {
+    echo "[perf] Delete $count vms"
+    for i in $(seq $count); do
+        limactl delete -f perf$i
+    done
 }
 
-host-to-vm-2() {
-    limactl start server &
-    limactl start client &
-    wait
-    limactl shell server sudo systemctl start iperf3.service
-    iperf3-darwin --client $(server_address) --time $TIME
-    limactl stop server
-    limactl stop client
+run() {
+    echo "[perf] Updating vms"
+    update_vms
+
+    echo "[perf] Starting vms"
+    start_vms
+
+    echo "[perf] Starting iperf3 service"
+    limactl shell perf1 sudo systemctl start iperf3.service >>perf.log
+
+    addr=$(server_address)
+    mkdir -p "$out_dir"
+
+    echo "[perf] Running $mode-$vm_type-host-to-vm-$count"
+    iperf3-darwin --client $addr --time $time --json > "$out_dir/$mode-$vm_type-host-to-vm-$count.json"
```

Comment:
Need to change the format to `{mode}-{vm_type}-{count}-{test}` to make it easier to work  with the filename later, and it also makes more sense (vm count neer vm type).

Also maybe it should be `{mode}-{vm_type}-{count}/{test}.json` to make this easier to work with.